src/wallet/wallet.cpp

```diff
@@ -4045,6 +4376,77 @@ string CWallet::SpendZerocoin(std::string &thirdPartyaddress, int64_t nValue, li
     return "";
 }
 
+
+/**
+ * @brief CWallet::SpendZerocoin
+ * @param thirdPartyaddress
+ * @param nValue
+ * @param denomination
+ * @param wtxNew
+ * @param coinSerial
+ * @param txHash
+ * @param zcSelectedValue
+ * @param zcSelectedIsUsed
+ * @return
+ */
+string CWallet::SpendMultipleZerocoin(std::string &thirdPartyaddress, std::vector<std::pair<int64_t, libzerocoin::CoinDenomination>> denominations, CWalletTx &wtxNew,
```

Comment:
Is it possible to change to `const std::vector<std::pair<int64_t, libzerocoin::CoinDenomination>>& denominations` instead? This will prevent a new vector instantiation, thus increase execution performance.

Reply:
Yes, a separate instance is also not necessary here.